packages/react-components/react-skeleton/src/contexts/SkeletonContext.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+const skeletonContext = React.createContext<SkeletonContextValue | undefined>(undefined);
```

Comment:
```suggestion
const SkeletonContext = React.createContext<SkeletonContextValue | undefined>(undefined);
```

nit: Let's use PascalCase for naming contexts